app/controllers/api/v2/compliance/arf_reports_controller.rb

```diff
@@ -84,12 +80,27 @@ def find_resource
           instance_variable_set("@arf_report", resource_scope.find(params[:id]))
         end
 
+        def find_resources_before_create
+          @asset = ForemanOpenscap::Helper::get_asset(params[:cname], params[:policy_id])
+          if !params[:openscap_proxy_url] && !params[:openscap_proxy_name]
```

Comment:
this now became hard failure, meaning older proxies without https://github.com/theforeman/smart_proxy_openscap/pull/60 won't be able to upload arf reports, correct? should we rather fallback to asset's host proxy in such case to remain compatible, just with the limitation for LB?